presto-main/src/main/java/io/prestosql/operator/EnforceSingleRowOperator.java

```diff
@@ -63,7 +61,7 @@ public OperatorFactory duplicate()
         }
     }
 
-    private static final Page SINGLE_NULL_VALUE_PAGE = new Page(1, new FixedWidthBlock(0, 1, EMPTY_SLICE, Optional.of(Slices.wrappedBooleanArray(true))));
+    private static final Page SINGLE_NULL_VALUE_PAGE = new Page(1, new ByteArrayBlock(1, Optional.of(new boolean[] {true}), new byte[1]));
```

Comment:
unrelated, but there seem to assumption that `EnforceSingleRowOperator` produces only single column (single scalar). Is it always the case @kokosing ?

Reply:
Yes. Nothing changed here.